pkg/manifest/os.go

```diff
@@ -737,6 +739,36 @@ func (p *OS) serialize() osbuild.Pipeline {
 		pipeline.AddStages(osbuild.GenFileNodesStages(p.Files)...)
 	}
 
+	// write modularity related configuration files
+	if len(p.moduleSpecs) > 0 {
+		pipeline.AddStages(osbuild.GenDNFModuleConfigStages(p.moduleSpecs)...)
```

Comment:
Looking at the code below, would it make sense to extend the `GenDNFModuleConfigStages()` (maybe a renamed version), to generate all of the relevant osbuild stages for a given `moduleSpecs`, including any `fsnode.File` that were created? The current version is, however, OK from my PoV.

Reply:
Let's keep it as is for now then and I'll note a followup to do that :)